src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.commands;
+
+import java.util.Comparator;
+
+import seedu.address.model.person.ReadOnlyPerson;
+
+/**
+ * Sorts all persons in the address book to the user.
+ */
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+    public static final String COMMAND_USAGE = COMMAND_WORD;
```

Comment:
There are probably better options to autocomplete `sort` command, like include both a dummy/default `field` parameter and a dummy/default `order` parameter.